pilot/pkg/networking/core/v1alpha3/envoyfilter/cluster_patch.go

```diff
@@ -35,7 +35,7 @@ func ApplyClusterMerge(pctx networking.EnvoyFilter_PatchContext, efw *model.Envo
 	c *cluster.Cluster, hosts []host.Name,
 ) (out *cluster.Cluster) {
 	defer runtime.HandleCrash(runtime.LogPanic, func(any) {
-		log.Errorf("clusters patch caused panic, so the patches did not take effect")
+		log.Errorf("clusters patch %s caused panic, so the patches did not take effect", efw.Namespace+"/"+efw.Name)
```

Comment:
Why not %s/%s with 2 arguments?

Reply:
No specific reason.  I can change it to that model

Reply:
nit